### PR TITLE
feat(storybook): updated html title

### PR DESCRIPTION
**Pull request by ayhandoslu**
**iframe.html** and **index.html** files in **storybook-static** folder (generated with `build-storybook`) contain `<title>Webpack App</title>` as default.

Added **webpackFinal** (for iframe.html) and **managerWebpack** (for index.html) into **.storybook/main.js** to override **HtmlWebpackPlugin**'s `title` option.

### CODE REVIEW

**Comment by muratcorlu** [comment]
Thank you for your contribution @ayhandoslu! 🎉

I'm having a look why Sonar job failed here...

File: .storybook/main.js [hunk 1]
```diff
@@ -16,4 +30,6 @@ module.exports = {
   },
   framework: '@storybook/web-components',
   staticDirs: ['../dist'],
+  webpackFinal: configureWebpack,
+  managerWebpack: configureWebpack,
```
Comment (muratcorlu) [inline .storybook/main.js:34]: Do we need both `webpackFinal` and `managerWebpack` here?

Comment (muratcorlu) [reply .storybook/main.js:34]: Ah you already described this in PR description, sorry.

**Review by muratcorlu — APPROVED** [review]

**Comment by ghengeveld** [comment]
Chromatic does not customize your webpack config. It invokes `build-storybook` to generate a static Storybook build, and uploads those files to S3. What you end up seeing published on Chromatic is 99% your static build, with some very small additions to handle embeds and such. The page title is not rewritten in any way. Can you run a static build yourself (`npm run build-storybook`) and verify the contents of `storybook-static/index.html`?

**Comment by muratcorlu** [comment]
Thank you for the clarification @ghengeveld! Now, when I check same url on chromatic I can see desired title there too. Weird but maybe It was a temporary cache issue.

Now I don't see any issues. Thank you all!

**Pull request merged**